src/TouristInfoCards.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import InfoCard from "./InfoCard";
+import manchesterImg from "./images/manchester.webp";
+import londonImg from "./images/london.jpeg";
+import glassgowImg from "./images/glasgow.jpeg";
+
+const TouristInfoCards = (props) => {
```

Comment:
If no props are being used inside of the component it doesnt need to have the props keyword here.